pkg/restapi/oidc/operations.go

```diff
@@ -42,6 +46,12 @@ type Config struct {
 	Storage    *StorageConfig
 	UIEndpoint string
 	TLSConfig  *tls.Config
+	Keys       *KeyConfig
+}
+
+type KeyConfig struct {
+	Auth [32]byte
+	Enc  [32]byte
```

Comment:
idem for key size in type here, it should be removed

Reply:
done